cmd/dep/ensure_test.go

```diff
@@ -1,13 +1,16 @@
-// Copyright 2016 The Go Authors. All rights reserved.
+// Copyright 2017 The Go Authors. All rights reserved.
```

Comment:
I think I glossed over this in the rebase. I created this file on my branch before this file was created in a recent merge.

https://github.com/golang/dep/commit/b8f7027092133c37c4503eb16a70660c3e571273

So when I added the copyright, I just put 2017 in it.

I can remove the change if you like.